update json plan docs with drift output (#31562)
hashicorp/terraform: website/docs/internals/json-format.mdx

    }
  ],

  // "resource_drift" is a description of the individual changes detected when
  // Terraform compared the most recent state to the prior saved state.
  "resource_drift": [
    {
        // "resource_drift" utilizes the same object structure as
        // "resource_changes".
    }
  ],

>   // "relevant_attributes" lists the sources of all values contributing to
>   // changes in the plan. "relevant_attributes" can be used to filter
>   // "resource_drift" to determine which external changes may have affected the
>   // plan result.

Reviewer: ```suggestion
  // "relevant_attributes" lists the sources of all values contributing to
  // changes in the plan. You can use "relevant_attributes" to filter
  // "resource_drift" and determine which external changes may have affected the
  // plan result.
```
